### PR TITLE
friendlier readme

### PR DESCRIPTION
**Pull request by Frijol**

### CODE REVIEW

**Comment by tcr** [comment]
Whoa, you took bit.ly/18Freadme to heart very quickly.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+[Join the conversation on our community chat:](https://tessel-slack.herokuapp.com/)  <img src="https://tessel-slack.herokuapp.com/badge.svg"></a>
 
 ## What is this project?
-Tessel is a completely open source and community-driven IoT and robotics development platform. It encompases development boards, hardware module add-ons, and the software that runs on them. Tessel is about designing intuitive and accessible hardware development experiences around the open web. Feel free to read more about our [mission](MISSION.md) and [governance model](GOVERNANCE.md). Please also read and adhere to our [code of conduct](https://github.com/tessel/project/blob/master/CONDUCT.md).
+Tessel is a completely open source and community-driven IoT and robotics development platform. It encompases development boards, hardware module add-ons, and the software that runs on them.
+
+### What's a Tessel?
+Our website [tessel.io](https://tessel.io) should give you a good idea of what we're making. You can see examples of things people have built on Tessel on our [projects page](https://tessel.io/projects).
+
+### What is this Tessel project all about?
```
Comment (tcr) [inline README.md]: Caps for `Project`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+Tessel is about designing intuitive and accessible hardware development experiences around the open web. Feel free to read more about our [mission](MISSION.md) and [governance model](GOVERNANCE.md). Please also read and adhere to our [code of conduct](https://github.com/tessel/project/blob/master/CONDUCT.md).
+
+### What can I find in this repo?
+This repository is for the organization and governance of the Tessel project as a whole. For code, see the [Tessel GitHub organization](https://github.com/tessel).
 
-This repository is for the organization and governance of the Tessel project as a whole. For code, see the [Tessel GitHub organization](https://github.com/tessel). 
+### What are the other important repos for Tessel?
```
Comment (tcr) [inline README.md]: I think "important repos" is less important than "start page", we should put the latter first.

**Comment by Frijol** [comment]
oh heh I didn't read it actually I just cringed watching someone try to understand our project from here

File: README.md [hunk 3]
```diff
@@ -1,14 +1,27 @@
 # <img src="https://cloud.githubusercontent.com/assets/80639/7736468/c78ac686-fef8-11e4-9931-cc3ef8fd37a0.png" width="600" alt="The Tessel Project">
 
-<a href="https://tessel-slack.herokuapp.com/">Join the conversation on our community chat:  <img src="https://tessel-slack.herokuapp.com/badge.svg"></a>
+Looking for the entry point to the Tessel Project? You've come to the right place.
```
Comment (tcr) [inline README.md]: Should we link to t2-start and t1-start here? I feel like linking people to "how to use Tessel" via Github is so hard

File: README.md [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
+
+[Join the conversation on our community chat:](https://tessel-slack.herokuapp.com/)  <img src="https://tessel-slack.herokuapp.com/badge.svg"></a>
 
 ## What is this project?
-Tessel is a completely open source and community-driven IoT and robotics development platform. It encompases development boards, hardware module add-ons, and the software that runs on them. Tessel is about designing intuitive and accessible hardware development experiences around the open web. Feel free to read more about our [mission](MISSION.md) and [governance model](GOVERNANCE.md). Please also read and adhere to our [code of conduct](https://github.com/tessel/project/blob/master/CONDUCT.md).
+Tessel is a completely open source and community-driven IoT and robotics development platform. It encompases development boards, hardware module add-ons, and the software that runs on them.
+
+### What's a Tessel?
```
Comment (tcr) [inline README.md]: "...should give you an overview of the hardware we build."

File: README.md [hunk 5]
```diff
@@ -31,18 +44,17 @@ If you have questions, please make a post on [our forums](https://forums.tessel.
 
 Get involved with our community in any way you are interested:
 
-* [Join us on Slack](https://tessel-slack.herokuapp.com/) &mdash; Collaboration and real time discussions.
+* **[Join us on Slack](https://tessel-slack.herokuapp.com/) &mdash; Collaboration and real time discussions (recommended - ask your questions here).**
```
Comment (tcr) [inline README.md]: s/recommended - /Recommended!/

File: README.md (hunk 2 above)
Comment (Frijol) [inline README.md]: Start page is up at the top now - more appropriate to redirect and assume most people landing here are for the project and/or dev work I think

**Comment by tcr** [comment]
lgtm

**Pull request merged**